src/components/TimelineNav/TimelineNav.module.css

```
@@ -160,7 +160,7 @@
     content: '';
     position: absolute;
     top: 0;
-    left: 0.75rem;
+    left: 0.75rem; /* 2 */
     height: 100%; /* 3 */
```

We're actually trying to move away from the numbers next to/above lines of styling and instead want to put the text of the comment next to the line it describes. Here's an example of what we're looking for:
```suggestion
    /**
     * This sets the height of the vertical line relative to the height of the current list item. We want a small visual
     * gap between the bottom of the line and the next bullet/icon.
     */
    height: 100%;
```
And then we would remove the `* 3)` comment on line 155.